bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/widgets/Event.java

```diff
@@ -357,13 +359,26 @@ public void setBounds (Rectangle rect) {
 	this.height = rect.height;
 }
 
+private int getDeviceZoom() {
```

Comment:
This should just be `getZoom()`, shouldn't it?

Reply:
Reverted changes for Event, since it is in common. @akoch-yatta has a separate PR for this.